build.sbt

```diff
@@ -126,6 +127,7 @@ lazy val commonJsSettings = Seq(
   jsEnv := new org.scalajs.jsenv.nodejs.NodeJSEnv(),
   // batch mode decreases the amount of memory needed to compile Scala.js code
   scalaJSLinkerConfig := scalaJSLinkerConfig.value.withBatchMode(isTravisBuild.value),
+  scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule)),
```

Comment:
Why is this change necessary?

Reply:
I honestly don't know, but the error message I got when running the tests without it pointed me to this. :smile: